### ISSUE TITLE
'r today' is not working properly.

### ISSUE THREAD

**Issue by Daeho-Son** [surrealroad/alfred-reminders#124]
https://user-images.githubusercontent.com/48791587/158303215-298e8c36-8b06-4676-88e6-5b9c35a83298.mov

<img width="1110" alt="스크린샷 2022-03-15 12 30 36" src="https://user-images.githubusercontent.com/48791587/158303432-01b21bb9-42e9-4d81-bfef-02eb31d9092a.png">

내일 = tomorrow

---------------------------
macOS Monterey ver 12.2.1
Macbook Air(M1, 2020)

Reminders for Alfred v75

**Comment by KngZhi** [comment]
same here

**Comment by kanapka94** [comment]
Find this code in workflow block named "r":

```js
if (d < now) {
  d.setDate(now.getDate() + 1);
}
```

and replace it with this:

```js
// If date is in the past, assume intended date is today

    if (d < now) {
      d.setDate(now.getDate());
      d.setHours(new Date().getHours())
      d.setMinutes(new Date().getMinutes())
    }
```

**Comment by kecinzer** [comment]
It's possible to set today without specify time?

**Comment by Christian-Martensson** [comment]
@kecinzer, did you find a solution for setting only today's date and no time?

**Comment by kecinzer** [comment]
> @kecinzer, did you find a solution for setting only today's date and no time?

No, I switched to Things 3.

**Comment by surrealroad** [comment]
> It's possible to set today without specify time?

I've been looking into a solution for this, there's probably a way to do it via chrono, but I can't see it. I don't want to create a lot of (more) spaghetti code to make it work.

**Comment by surrealroad** [comment]
fixed via b04fe89647fe90b849c8f8ad5c3463cbbea9a0db
